### PR TITLE
Added more contests, precincts, and polling locations to sample feed

### PR DESCRIPTION
**Pull request by jdmgoogle**
More additions to the sample feed, including

1. A judicial retention election;
2. An all-mail-in precinct; and
3. A combination drop-box/early vote site.

I also cleaned up several dangling or invalid xs:IDREF values.

### CODE REVIEW

**Comment by jdmgoogle** [comment]
This addresses more requests found in #179.

Pinging @jktomer in particular since I'm sure many jurisdictions will base their feeds on this one, meaning he'll be the most prominent consumer of this data. Given that he sits within arm's reach of me I don't want to give him any particular motivation to strangle me for inspiring poor quality feeds. :)

**Comment by jungshadow** [comment]
@jdmgoogle I'm all for keeping this a "Strangle-Free Zone," but if you need me [to suit-up for backup](http://bukk.it/meanwhile.jpg)... ;)

Changes look fine to me, FWIW.

**Comment by jdmgoogle** [comment]
I've definitely noticed that there's a relationship between how late it is and the probability that @jungshadow will break out a random gif or jpeg in a comment ....

(And thanks.)

**Comment by jungshadow** [comment]
@jdmgoogle It's nice to know we speak the same language :+1:

File: sample_feed.xml [hunk 1]
```diff
@@ -2052,11 +2174,24 @@
...
+    <NonHouseAddress>
+      <City>Charlottesville</City>
+      <State>VA</State>
+      <StreetName>CHAPEL HILL RD</StreetName>
+      <Zip>22901</Zip>
+    </NonHouseAddress>
+    <OddEvenBoth>both</OddEvenBoth>
+    <PrecinctId>pre99999</PrecinctId>
+    <StartHouseNumber>0</StartHouseNumber>
+    <EndHouseNumber>0</EndHouseNumber>
```
Comment (jktomer) [inline sample_feed.xml]: Don't make me punch Aaron in the face all over again.

**Comment by jktomer** [comment]
LGTM except for ss999999.{Start,End}HouseNumber

**Comment by pstenbjorn** [comment]
And StreetSuffix is contained in the StreetName, bad source data.  And I am part of the source.

**Comment by jdmgoogle** [comment]
> LGTM except for ss999999.{Start,End}HouseNumber

What do you recommend? Leaving it blank?

**Comment by jktomer** [comment]
> > LGTM except for ss999999.{Start,End}HouseNumber
> What do you recommend? Leaving it blank?

Talked offline. It might be nice to add an "IsWholeStreet" alternative to start and end house number and leave those fields out in such cases. I'd be fine either with that or with changing this to [1, 99999].

**Comment by jdmgoogle** [comment]
I'm going to submit this as-is, clean up a few more items WRT 1622.2, and then add an `IsWholeStreet` element to `StreetSegment`.

**Pull request merged**